openidm-zip/src/main/resources/db/h2/scripts/openidm.sql

```diff
@@ -0,0 +1,482 @@
+CREATE INDEX IF NOT EXISTS fk_configobjects_objecttypes ON openidm.configobjects (objecttypes_id);
+
+-- -----------------------------------------------------
+-- Table openidm.configobjectproperties
+-- -----------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS openidm.configobjectproperties (
+  configobjects_id BIGINT NOT NULL,
+  propkey VARCHAR(255) NOT NULL,
+  proptype VARCHAR(255) DEFAULT NULL,
+  propvalue TEXT,
+  CONSTRAINT fk_configobjectproperties_configobjects FOREIGN KEY (configobjects_id) REFERENCES openidm.configobjects (id) ON DELETE CASCADE ON UPDATE NO ACTION
+);
+
+CREATE INDEX IF NOT EXISTS fk_configobjectproperties_configobjects ON openidm.configobjectproperties (configobjects_id);
+-- CREATE INDEX IF NOT EXISTS idx_configobjectproperties_prop ON openidm.configobjectproperties (propkey,propvalue);
+
+-- -----------------------------------------------------
+-- Table openidm.relationships
+-- -----------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS openidm.relationships (
+  id BIGSERIAL NOT NULL,
+  objecttypes_id BIGINT NOT NULL,
+  objectid VARCHAR(255) NOT NULL,
+  rev VARCHAR(38) NOT NULL,
+  fullobject JSON,
+  PRIMARY KEY (id),
+  CONSTRAINT fk_relationships_objecttypes FOREIGN KEY (objecttypes_id) REFERENCES openidm.objecttypes (id) ON DELETE CASCADE ON UPDATE NO ACTION,
+  CONSTRAINT idx_relationships_object UNIQUE (objecttypes_id, objectid)
+);
+
+-- CREATE INDEX IF NOT EXISTS idx_json_relationships_first ON openidm.relationships ( json_extract_path_text(fullobject, 'firstId'), json_extract_path_text(fullobject, 'firstPropertyName') );
+-- CREATE INDEX IF NOT EXISTS idx_json_relationships_second ON openidm.relationships ( json_extract_path_text(fullobject, 'secondId'), json_extract_path_text(fullobject, 'secondPropertyName') );
+-- CREATE INDEX IF NOT EXISTS idx_json_relationships ON openidm.relationships ( json_extract_path_text(fullobject, 'firstId'), json_extract_path_text(fullobject, 'firstPropertyName'), json_extract_path_text(fullobject, 'secondId'), json_extract_path_text(fullobject, 'secondPropertyName') );
+
+-- -----------------------------------------------------
+-- Table openidm.relationshipproperties (not used in postgres)
+-- -----------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS openidm.relationshipproperties (
+  relationships_id BIGINT NOT NULL,
+  propkey VARCHAR(255) NOT NULL,
+  proptype VARCHAR(32) DEFAULT NULL,
+  propvalue TEXT,
+  CONSTRAINT fk_relationshipproperties_relationships FOREIGN KEY (relationships_id) REFERENCES openidm.relationships (id) ON DELETE CASCADE ON UPDATE NO ACTION
+);
+CREATE INDEX IF NOT EXISTS fk_relationshipproperties_relationships ON openidm.relationshipproperties (relationships_id);
+-- CREATE INDEX IF NOT EXISTS idx_relationshipproperties_prop ON openidm.relationshipproperties (propkey,propvalue);
```

Comment:
Same.

Reply:
changed propvalue data type to VARCHAR(65535)